browser/ui/webui/brave_wallet/wallet_page_ui.cc

```diff
@@ -66,6 +66,14 @@ WalletPageUI::WalletPageUI(content::WebUI* web_ui)
   webui::SetupWebUIDataSource(source, base::span(kBraveWalletPageGenerated),
                               IDR_WALLET_PAGE_HTML);
   source->AddString("braveWalletLedgerBridgeUrl", kUntrustedLedgerURL);
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::FrameAncestors,
+      base::JoinString(
+          {"frame-ancestors", kBraveUIWalletPageURL, kBraveUIWalletPanelURL},
+          " ") +
+          ";");
```

Comment:
Do we need this for wallet page and panel? Maybe that should be `none` so we just don't allow them being a subframe